spec/inverted-index-helper.js

```diff
@@ -1,25 +1,55 @@
-class invertedIndexHelper {
+/**
+ * InvertedIndexHelper
```

Comment:
### Critical

Why do you have duplicate copies of `InvertedIndexHelper and InvertedIndex` in the `spec` folder,  you should only have your tests in that directory.